packages/cli/src/util/dev/builder.ts

```diff
@@ -439,6 +437,8 @@ export async function getBuildMatches(
     for (const file of files) {
       src = relative(cwd, file);
 
+      const entrypoint = mapToEntrypoint.get(src) || src;
```

Comment:
Is this just a formatting change or is there something else depending on it now?

Reply:
It was a change to make `entrypoint` be calculated _before_ the update to `src` to strip the output directory. It was necessary to do it like this to make `@vercel/static` work without changes.